src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -105,6 +105,35 @@ inline bool BinaryBroadcastMulStorageType(const nnvm::NodeAttrs& attrs,
   return dispatched;
 }
 
+inline bool BinaryBroadcastAddStorageType(const nnvm::NodeAttrs& attrs,
+                                          const int dev_mask,
+                                          DispatchMode* dispatch_mode,
+                                          std::vector<int>* in_attrs,
+                                          std::vector<int>* out_attrs) {
+  CHECK_EQ(in_attrs->size(), 2U);
+  CHECK_EQ(out_attrs->size(), 1U);
+  const int lhs_stype = in_attrs->at(0);
+  const int rhs_stype = in_attrs->at(1);
+  int& out_stype = out_attrs->at(0);
+  bool dispatched = false;
+  // For GPU, directly fallback
+  const auto dispatch_ex = (dev_mask == mshadow::gpu::kDevMask)? DispatchMode::kFComputeFallback :
```

Comment:
The GPU implementation should be fucntional, right? Can we add that? Later we can keep improving performance

Reply:
I'll test the functionality of the GPU version and if that works properly I'll enable that.